CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Fixed
+- We upgraded minimum sdk version to 33
```

Comment:
We didn't upgrade minimum sdk version in this PR

Reply:
My bad, done 👍